pyatv/const.py

```diff
@@ -434,3 +434,23 @@ class FeatureName(Enum):
 
     TextSet = 54
     """Replace text in virtual keyboard."""
+
+    TouchGesture = 63
+    """Touch gesture from given coordinates and duration."""
+
+    TouchEvent = 64
+    """Touch event to given coordinates."""
+
+    TouchClick = 65
+    """Touch click command."""
+
+    TouchHold = 66
+    """Hold touch command."""
+
+
+class HidEventMode(Enum):
```

Comment:
I think we sold call this something like `TouchAction` as `HID` is just an implementation detail in this context. Could change to something else in the future.

Reply:
Do we know what 2 is btw?

Reply:
I have not seen it as of now. However, I believe that "Hold" ought to be renamed to "Drag" to not confuse it with "press and *hold*"

Reply:
I didn't see any message with 2, so only guess at this state

Reply:
Renamed